File: Unit13/src/com/practicaljava/lesson23/model/QuoteModel.java
package com.practicaljava.lesson23.model;


/**
 * @author Svyatoslav Strygin <svyatoslav.strygin@gmail.com>
 */
public class QuoteModel {
	
	public String symbol;
	public int quantity;
	public double price;
	
	
	public QuoteModel(String symbol, int quantity, double price){
		this.symbol = symbol;
		this.quantity = quantity;
		this.price = price;
	}
	
	public void printQuote(){
		System.out.println("symbol: " + symbol + " quantity: " + quantity + " price: " + price);
	}

}

File: Unit14/lesson27/WebContent/scripts/QuoteModel.js
/**
 * Observable Quote Model
 */
var ObservableQuoteModel = function(quote) {
	this.symbol = ko.observable(quote.symbol);
	this.price = ko.observable(quote.price);
	this.quantity = ko.observable(quote.quantity);
	this.textColor = ko.observable(quote.textColor);
};